docker/accumulo/entrypoint.sh

```diff
@@ -32,6 +34,14 @@ if [ "$1" = "accumulo" ] && [ "$2" = "master" ]; then
 		fi
 	fi
 
+    # Try to find desired root password from accumulo.properties (accumulo 2)
```

Comment:
Indenting wrong

Reply:
For the record I had used spaces whereas the original file used tabs, hence why I couldn't see a difference locally